hackathon.html

```diff
@@ -0,0 +1,102 @@
+<script type="text/javascript" src="js/handlebars-v4.0.10.js"></script>
+<div class="container">
+<div id="nixcon-logo">
+  <img src="/images/nixos-logo.png" class="responsive-img nixos-logo" width="300" style="margin-top: 2em; margin-bottom: 0em;" />
+</div>
+
+  <div class="row">
+    <div class="col s12 m12">
+      <h1 class="header center grey-text text-darken-2">NixCon Hackathon</h1>
+      <h3 class="header center grey-text text-darken-2">Part of the <a href="./">NixCon 2017</a></h3>
+    </div>
+  </div>
+  <br/>
+</div>
+
+<div class="container">
+  <div class="row">
+    <div class="col s12 m12">
+      <h3 class="header center grey-text text-darken-2">Topics</h3>
+    </div>
+  </div>
+  <div class="row" id="topics">
+  </div>
+  <div class="row">
+    <div class="col s12 m12">
+        <h5>Want to propose your own topic?</h5>
+        You can propose new topics via the following workflow:
+        <ol>
+            <li>fork the page repository <a href="https://github.com/cko/nixcon2017">cko/nixcon2017</a></li>
+            <li>add your topic to the file <tt>./hackathon-topics.json</tt>
+                <ul>
+                    <li>the skill level range goes from 1 (Beginner) to 3 (Expert)</li>
+                </ul>
+            </li>
+            <li>create a pull request</li>
+        </ol>
+    </div>
+  </div>
+</div>
+<script type="text/javascript" src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
+<script>
+ $( document ).ready(function() {
+    function skillLevelToHTML(lvl) {
+        switch(lvl) {
+            case 1: return "Beginner";
+            case 2: return "Advanced";
+            case 3: return "Expert";
+            default: return "?";
+        }
+    }
+
+    function skillLevelsToHTML(lvl1, lvl2) {
+        if (lvl1 === lvl2) {
+            return skillLevelToHTML(lvl1);
+        }
+        return skillLevelToHTML(lvl1) + " - " + skillLevelToHTML(lvl2)
+    }
+
+    $.getJSON( "./hackathon-topics.json", function( data ) {
```

Comment:
Please don't include dummy data in the page

Reply:
I would remove the dummy data once the first topic was proposed.

Reply:
Dummy data will no longer be rendered.